covidWatch/home.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>COVIDWatch Homepage</title>
+</head>
+<body>
+	<a href="#"><img src="images/logo.png"></a>
+  		<a href="#">HOME</a>
```

Comment:
You could use a nav here for the links: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_nav

Reply:
Done